ros2launch_security_examples/test/test_fake_imu_launch_with_security.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import unittest
+
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch.actions import SetEnvironmentVariable
+
+import launch_testing
+import launch_testing_ros
+
+g_this_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def generate_test_description():
+    launch_description = LaunchDescription()
+
+    launch_description.add_action(
+        # bare minimum formatting for console output matching
+        SetEnvironmentVariable('RCUTILS_CONSOLE_OUTPUT_FORMAT', '{message}')
+    )
+    launch_description.add_action(
+        ExecuteProcess(
+            cmd=[
+                'ros2', 'launch',
+                os.path.join(g_this_dir, '..', 'launch', 'fake_imu.launch.xml'),
+                '--secure',
+            ],
```

Comment:
I think it would be good to put a comment here explaining why you have to do things this way and not use the builtin `IncludeLaunchDescription` as you do in the non-security version.

Reply:
This comment below it explains a bit, but I can put one here too:

https://github.com/osrf/ros2launch_security/pull/4/files#diff-71305fd43c410f758e5a1ece15052f54aa928231021fd86a401b2a5f4bff0309R72-R76

73675e6